src/pages/networks/NetworkList.tsx

```diff
@@ -12,7 +13,7 @@ import { useQuery } from "@tanstack/react-query";
 import { queryKeys } from "util/queryKeys";
 import Loader from "components/Loader";
 import { useNavigate, useParams } from "react-router-dom";
-import EmptyState from "components/EmptyState";
```

Comment:
Please also remove the local component in `components/EmptyState.tsx`

Reply:
I see, makes sense. Be careful with changes that should not be committed. One way is to add a `//todo: do not commit` comment above the change, so it won't accidentally make it into a commit.

Reply:
That's a neat trick thanks :)

Reply:
Also, no need to push to remote any little and temporary work-in-progress. You can keep the changes locally, and push them only once at the end, unless you need to test them on the demo server, but it doesn't seem the best way to proceed since each build takes more than 5 minutes to complete.

Reply:
You're right, I'll squash the ones here and the ones I have locally and push.